contracts/seed/Seed.sol

```diff
@@ -101,7 +102,8 @@ contract Seed {
       * @param _softHardThresholds    Array containing two params:
                                         - the minimum funding token collection threshold in wei denomination.
                                         - the highest possible funding token amount to be raised in wei denomination.
-      * @param _price                 price of a SeedToken, expressed in fundingTokens, with precision of 10**18
+      * @param _priceSeedTokenDecimal price of a SeedToken, expressed in fundingTokens, in units of funding token
+                                       And Seed Token Decimal
```

Comment:
_priceSeedTokenDecimals (plural)

This is confusing.  Is it the number of decimals, or is it the price?  Can we be totally clear about this in the naming and the comment?  Note we already have _seedTokenDecimals

If this is an array, can you please say that in the comment and indicate clearly what is [0] and what is [1]?